### PR TITLE
Using `DIR_DEFAULT_DOWNLOADS_SAFE` 

### PR DESCRIPTION
**Pull request by darkdh**
when DIR_DEFAULT_DOWNLOADS got home directory or desktop

fix https://github.com/brave/browser-laptop/issues/13229
fix https://github.com/brave/browser-laptop/issues/13255

Auditors: @bridiver, @bsclifton

### CODE REVIEW

File: atom/browser/api/atom_api_dialog.cc [hunk 1]
```diff
@@ -86,6 +87,23 @@ struct Converter<file_dialog::DialogSettings> {
 
 namespace {
 
+// Consider downloads 'dangerous' if they go to the home directory on Linux and
+// to the desktop on any platform.
+bool DownloadPathIsDangerous(const base::FilePath& download_path) {
```
Comment (bridiver) [inline atom/browser/api/atom_api_dialog.cc]: cc @jumde - this will conflict with your PR I think and the safe_browser changes should take priority

Comment (bridiver) [reply atom/browser/api/atom_api_dialog.cc]: in your branch please update this to use download_prefs.cc version of DownloadPathIsDangerous

Comment (bridiver) [reply atom/browser/api/atom_api_dialog.cc]: or remove the whole patch if this is no longer necessary

Comment (darkdh) [reply atom/browser/api/atom_api_dialog.cc]: @jumde please remove this changes in https://github.com/brave/muon/pull/409 once you rebased. Thank you

Comment (darkdh) [reply atom/browser/api/atom_api_dialog.cc]: same functionality is covered in https://chromium.googlesource.com/chromium/src/+/533c62725be56fe49be4ab548e7dd7061b1c977c/chrome/browser/download/download_prefs.cc#154

**Pull request merged**